spring-cloud-config-server/src/main/java/org/springframework/cloud/config/server/environment/vault/SpringVaultEnvironmentRepositoryFactory.java

```diff
@@ -49,23 +37,20 @@
 
 	private final EnvironmentWatch watch;
 
-	private final ConfigTokenProvider tokenProvider;
+	private final SpringVaultClientConfiguration clientConfiguration;
 
 	public SpringVaultEnvironmentRepositoryFactory(
```

Comment:
I think we should add a new constructor rather than change the existing one so we avoid breaking anyone

Reply:
This class and its constructor are brand new, didn't exist before yesterday: https://github.com/spring-cloud/spring-cloud-config/commit/5e33794332815290b75af47874487db69f47ccc0.

Reply:
Ah ok :) . I didnt get a chance to look at it